src/Loading/Plugins/babylon.babylonFileLoader.ts

```diff
@@ -680,7 +680,16 @@
 
                 // Environment texture		
                 if (parsedData.environmentTexture !== undefined && parsedData.environmentTexture !== null) {
-                    scene.environmentTexture = CubeTexture.CreateFromPrefilteredData(rootUrl + parsedData.environmentTexture, scene);
+                    if (parsedData.environmentTextureType && parsedData.environmentTextureType === "BABYLON.HDRCubeTexture") {
+                        var hdrSize:number = (parsedData.environmentTextureSize) ? parsedData.environmentTextureSize : 128;
+                        var hdrTexture:BABYLON.HDRCubeTexture = new BABYLON.HDRCubeTexture(rootUrl + parsedData.environmentTexture, scene, hdrSize);
+                        if (parsedData.environmentTextureRotationY) {

```

Comment:
should we not apply rotation in case of prefiltered texture as well ?

Reply:
Would it not help if ppl wanted to rotate the environment lighting with prefiltered data ?

We added the support for rotationY in cubeTexture as well for this.